crates/forge-runner/src/runtime.rs

```diff
@@ -150,6 +195,19 @@ impl<Extension: ExtensionLogic> HintProcessorLogic for ExtendedRuntime<Extension
     }
 }
 
+impl<Handler, Runtime: SyscallPtrAccess + HintProcessor> SyscallPtrAccess
+    for RuntimeExtension<Handler, Runtime>
+{
+    fn get_mut_syscall_ptr(&mut self) -> &mut Relocatable {
+        let val: &mut _ = self.extended_runtime.get_mut_syscall_ptr();
```

Comment:
why not just `&mut self.extended_runtime.get_mut_syscall_ptr()?`

Reply:
Done